components/Banner/examples/resources/BannerAssets.xcassets/banner-email.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "ic_email_black_1x_ios_24dp.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "ic_email_black_2x_ios_24dp.png",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "ic_email_black_3x_ios_24dp.png",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
```

Comment:
If possible, please make use of an existing icon rather than adding a new asset to the examples.

Reply:
The other assets are included in the MDCCatalog/MDCDragon. This change was made to address a previous code review comment (https://github.com/material-components/material-components-ios/pull/6830#discussion_r264481238) to add assets to example assets folder. Not sure if there is a convention here?

Reply:
Agreed - the catalog assets aren't available to the examples targets unless linked against the catalog.